modules/tinymce/src/core/main/ts/fmt/FormatUtils.ts

```diff
@@ -9,6 +9,49 @@ import * as NodeType from '../dom/NodeType';
 import * as Whitespace from '../text/Whitespace';
 import { BlockFormat, Format, FormatAttrOrStyleValue, FormatVars, InlineFormat, MixedFormat, SelectorFormat } from './FormatTypes';
 
+const internalAttributesPrefixes = [
+  'data-ephox-',
+  'data-mce-',
+  'data-alloy-',
+  'data-snooker-',
+  '_',
+  'aria-'
```

Comment:
A lot of Aria attributes have `aria-` prefixes. I don't think they are TinyMCE specific

Reply:
But does it matter if they're removed?

I think I agree, this is new and it probably isn't a good idea.

- [ ] remove `aria-`

Reply:
Ah, this probably hung around from an earlier bit, so I'll remove it.